### PR TITLE
fix: missing fallback translation when no locale available

### PR DESCRIPTION
**Pull request by Fabio286**
Hi dev,
today i discovered the existence of this translation package for conventional-commit-types with an exception from my Vistual Studio Code installation.

This PR should fix this bug by providing english as fallback localization when a there is no specific localization.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
-import { readFileSync } from 'fs';
+import { readFileSync, readdirSync } from 'fs';
 import { resolve } from 'path';
 
 export function conventionalCommitsTypes(locale?: string) {
-  const fileName = locale ? `${locale}.json` : 'en.json';
+  const localizationFiles = readdirSync(resolve(__dirname, `../locale/`));
+  const fileName =
```
Comment (yi-Xu-0100) [inline src/index.ts:6]:
```suggestion
  const fileName = localizationFiles.includes(`${locale}.json`) ? `${locale}.json` : 'en.json';
```

It can deal with the `undefined` and empty string input.

File: src/index.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+    locale && localizationFiles.includes(`${locale}.json`) ? `${locale}.json` : 'en.json';
```
Comment (yi-Xu-0100) [inline src/index.ts:7]:
```suggestion
  const fileName = localizationFiles.includes(`${locale}.json`) ? `${locale}.json` : 'en.json';
```
It can deal with the undefined and empty string input.

File: tests/index.test.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { conventionalCommitsTypes } from '../src/index';
 describe('conventionalCommitsTypes() function test', () => {
   it.each([
     ['en.json', undefined],
+    ['en.json', 'unknown-locale'],
```
Comment (yi-Xu-0100) [inline tests/index.test.ts:8]:
```suggestion
    ['en.json', ''],
    ['en.json', 'unknown-locale'],
```

Add test for the empty string input.

**Comment by yi-Xu-0100** [comment]
@Fabio286 I will do the change by myself and publish it quickly. Thanks for your contribution. 👍
